main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+	"time"
+
+	"github.com/google/go-containerregistry/pkg/authn"
+	"github.com/google/go-containerregistry/pkg/crane"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/empty"
+	"github.com/google/go-containerregistry/pkg/v1/mutate"
+	"github.com/google/go-containerregistry/pkg/v1/tarball"
+)
+
+func build(diskPath string) (v1.Image, error) {
+	// TODO: Should be in /disk directory
+	// TODO: Should be 107 qemu user
```

Comment:
AFAIK both aren't hard requirements, did something fail down the line when not setting it? AFAIR KubeVirt searches for disk images inside a container image and if it finds only one it is used regardless of the path.

Reply:
This is not what documented in the docs, e.g. "Disks must be placed into the /disk directory inside the container." [1].

[1] https://kubevirt.io/user-guide/virtual_machines/disks_and_volumes/#containerdisk-workflow-example

Reply:
Done.